scheduler/feasible.go

```diff
@@ -212,6 +212,7 @@ func (h *HostVolumeChecker) hasVolumes(n *structs.Node) bool {
 					capOk = true
 					break
 				}
+				return false
```

Comment:
We shouldn't return early here -- a volume can have multiple requested capabilities and we only need one of them to match.